repos/system_upgrade/el7toel8/actors/peseventsscanner/tests/unit_test.py

```diff
@@ -1,4 +1,4 @@
-import os
+import os  # noqa: F401
```

Comment:
test_pes_data_not_found should change instead of:
```python
monkeypatch.setattr('os.path.isfile', file_not_exists)
```
to
```python
monkeypatch.setattr(os.path, 'isfile', file_not_exists)
```

But I might be mistaken about this one, also since there's no other use, I'd even go with import os.path to make clear that os.path is what is wanted.